.github/workflows/new-app-review/validators/dependencies.ts

```diff
@@ -0,0 +1,48 @@
+import { spawn } from 'child_process';
+import type { ValidatorOptions, ValidatorResult, PullRequestFile } from '../../types';
+
+export const validate = async (_options: ValidatorOptions, newAppDir: string, _files: PullRequestFile[]): Promise<ValidatorResult> => {
+  return new Promise((resolve, reject) => {
+    const command = `npm outdated --json`;
```

Comment:
would doing an `npm audit` be valuable too?

Reply:
great call, took a stab at this here 3b3a438adc4e93fbf570a83f0f7456b14dea53da just need to test it end to end really quick